lightning/src/routing/scorer.rs

```diff
@@ -0,0 +1,51 @@
+// This file is Copyright its original authors, visible in version control
+// history.
+//
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! Utilities for scoring payment channels.
+//!
+//! [`Scorer`] may be given to [`get_route`] to score payment channels during path finding when a
+//! custom [`routing::Score`] implementation is not needed.
+//!
+//! # Example
+//!
+//! TODO: Fill in example once interface has been finalized.
```

Comment:
I think we're good to fill this in?

Reply:
Done. Not much to it yet, but could expand if we add more configurations. Also, may want to have an example with `InvoicePayer` for event handling once that is in. Or at least reference those docs.